src/operators.jl

```diff
@@ -442,3 +442,43 @@ This is equivalent to [`induced_subgraph`](@ref)`(g, neighborhood(g, v, d, dir=d
 with respect to `v` (i.e. `:in` or `:out`).
 """
 egonet(g::AbstractGraph, v::Integer, d::Integer; dir=:out) = g[neighborhood(g, v, d, dir=dir)]
+
+
+"""
+    merge_vertices!(g::Graph, vs::Array{Int})
+
+Combines vertices specified in `vs` into single vertex whose
+index will be the lowest value in `vs`.
+
+All edges running to vertices in `vs` will connect to new
+merged vertex.
+
+Returns an Array{Int} with new vertex values for
+"""
+
+function merge_vertices!(g::Graph, vs::Array{Int})
```

Comment:
`vs` should either be untyped or be a `Vector{T}` where `T <: Integer` since vertices don't have to be `Int`s.